### PR TITLE
Flows API v2

### PR DESCRIPTION
**Pull request by rowanseymour**
Since we have a definitions endpoint which handles (potentially expensive) flow definition fetching, this endpoint does not include flow definition

Adds UUIDs to flow labels for consistency with message labels, groups etc.

### CODE REVIEW

File: temba/api/v2/serializers.py [hunk 1]
```diff
@@ -220,6 +220,23 @@ class Meta:
...
+    archived = serializers.ReadOnlyField(source='is_archived')
+    labels = serializers.SerializerMethodField()
+    expires = serializers.ReadOnlyField(source='expires_after_minutes')
+    runs = serializers.SerializerMethodField()
+
+    def get_labels(self, obj):
+        return [{'uuid': l.uuid, 'name': l.name} for l in obj.labels.all()]
+
+    def get_runs(self, obj):
+        return {'completed': obj.get_completed_runs(), 'expired': obj.get_expired_runs()}
```
Comment (nicpottier) [inline temba/api/v2/serializers.py]: Should probably include all exit types here no? So interrupted as well?

**Comment by nicpottier** [comment]
Kinda sorta related, but we should change the flow edit/read page to have a UUID in the URL. All our endpoints require UUIDs to do stuff with flows but we only expose that UUID in exports and in the API which is a pain for users.

File: temba/api/v2/serializers.py (hunk 1 above)
Comment (rowanseymour) [inline temba/api/v2/serializers.py]: Yeah - have added that now

**Comment by ericnewcomer** [comment]
yarp

**Pull request merged**